apps/storybook/stories/hooks/Theme.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta } from "@storybook/addon-docs";
+import {DocsHeading, Feedback} from "../templates";
+import {Source} from "@storybook/blocks";
+import {ThemeStrings} from "./srcThemeStrings";
+
+<Meta title="Hooks/useTheme" />
+
+# useTheme
+
+**useTheme** er en tilpasset hook som brukes for å hente temaobjektet fra kontekst.
+
+```jsx
+import { useTheme } from "@kvib/react";
+```
+
+<Feedback component="useTheme"/>
+
+<DocsHeading>Return value</DocsHeading>
+**useTheme** returnerer temaobjektet.
```

Comment:
Her hadde det vært nice med en div som skifter farge avhengig av temaet eller noe slikt, bare for å vise litt mer hvordan dette fungerer.